draft-ietf-quic-multipath.md

```diff
@@ -528,6 +528,22 @@ proposed and implemented, notably for Multipath TCP. A companion draft
 {{I-D.bonaventure-iccrg-schedulers}} provides several general-purpose
 packet schedulers depending on the application goals.
 
+# Recovery
+
+Simmultaneous use of multiple paths enables new techniques to cope with packet losses.
+In {{QUIC-TRANSPORT}}, a lost packet can either be abandoned or retransmitted over the same path once it has been
+detected as lost. However, when multiple paths are available for the retransmission
+QUIC implementation can use different strategies to cope with losses.
+A simple strategy is to retransmit a lost frame over the same path.
+Another straegy would be to use the best path, e.g. in terms of latency,
+or use a dedicated path to send retransmissions. Depending on the
+appliaction needs another strategy could also be to simultaneously
+retransmit a lost frame over its original path and another one,
+however, this is not recommended for general purpose use due to
+the overhead but could be particualy beneficial for certain kind
+of data or frames. This document does not preclude a specific strategy,
+however, more detailed specification of any such stragtegy is out of scope.
```

Comment:
I think this might make the text more easy to read.
```suggestion
Possible strategies include: a) retransmitting lost frames over the
same path, b) retransmitting lost frames on a different or
dedicated path, and c) duplicate lost frames on several paths (not
recommended for general purpose use due to the network
overhead). While this document does not preclude a specific
strategy, more detailed specification is out of scope.
```